tests/smoke.runner.js

```diff
@@ -140,11 +140,11 @@ const jasmineTestrunner = async () => {
             'expect(number).toBe(number)',
             'expect(object).toEqual(object)',
             'expect(object).toBeFalse(boolean)',
-            'expect(object).toHaveTitle(object)',
-            'expect(object).toHaveTitle(object)',
-            'expect(object).toHaveUrl(object)',
-            'expect(object).toHaveUrl(object)',
-            'expect(string).toHaveTitle(object)',
+            'expect(object).toHaveTitle(function)',
+            'expect(object).toHaveTitle(function)',
+            'expect(object).toHaveUrl(function)',
+            'expect(object).toHaveUrl(function)',
+            'expect(string).toHaveTitle(function)',
```

Comment:
Why are these assertions changing?

Reply:
To be honest... I'm not sure. I've to investigate it.

Reply:
@christian-bromann I now have an explanation for this. I've made it so that `global.browser` and `browser` included by importing/requiring from `@wdio/globals` are consistent. If you take a look at this test in main stream https://github.com/webdriverio/webdriverio/blob/main/packages/wdio-globals/tests/index.test.ts#L6, you'll ralise that browser is a function from the import perspective. I can make it look like and `object` or like a `function`, but not both of them at the same time :-).